app/models/shipit/pull_request.rb

```diff
@@ -108,6 +112,7 @@ def reject!(reason)
     def reject_unless_mergeable!
       return reject!('merge_conflict') if merge_conflict?
       return reject!('ci_failing') unless all_status_checks_passed?
+      return reject!('timedout') if timedout?
```

Comment:
Yes. Worst case 1h and 59 seconds. Unless there is an outage or something and crons or background queue is broken.

Anyway I'm about to submit another PR so that the job is triggered more often than once per minute.